src/conn.c

```diff
@@ -3117,6 +3117,9 @@ _processUrlString(natsOptions *opts, const char *urls)
     char        *ptr         = NULL;
     int         len;
 
+    if (urls == NULL)
+        return NATS_INVALID_ARG;
```

Comment:
If we think it is better to have this fail, then I would call:
```
return nats_setDefaultError(NATS_INVALID_ARG);
```
so that it updates the stack and we know where the error comes from, or even better:
```
return nats_setError(NATS_INVALID_ARG, "%s", "urls argument cannot be NULL");
```
which would produce this kind of trace:
```
Error: 16 - Invalid Argument - (conn.c:3121): urls argument cannot be NULL
Stack: (library version: 2.3.0)
  01 - _processUrlString
  02 - natsConnection_ConnectTo
```